### PR TITLE
[FEAT] Set-Up

### PR DESCRIPTION
**Pull request by hannahjavs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+  },
   "dependencies": {
-    "enzyme": "^3.3.0",
-    "jest": "^23.0.1",
     "material-ui": "^1.0.0-beta.47",
+    "mobx": "^4.2.1",
     "mobx-react": "^5.1.2",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
     "ts-loader": "^3.5.0"
```
Comment (bradzacher) [inline package.json]: `ts-loader` should be a dev dependency

File: package.json [hunk 2]
```diff
@@ -1,12 +1,29 @@
 {
+  "name": "react-webpack-setup",
```
Comment (bradzacher) [inline package.json]: That's not your repo name!

**Review by bradzacher — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (hannahjavs) [reply package.json]: 🍻

Comment (hannahjavs) [reply package.json]: No need to shout...Maybe I want it to be my repo name!....................

Comment (bradzacher) [reply package.json]: I WASN'T SHOUTING, YET

**Review by bradzacher — APPROVED** [review]

**Comment by bradzacher** [comment]
next time use the dropdown arrow and select "squash and merge"

image

To make it easy you can disable the other merge types in the repository settings:
image

**Comment by hannahjavs** [comment]
I hovered over that...It was a gamble.......I lost.

**Pull request merged**